python/ray/serve/batching.py

```diff
@@ -306,6 +342,7 @@ async def __call__(self, request: Request):
             one call to the underlying function.
         batch_wait_timeout_s: the maximum duration to wait for
             `max_batch_size` elements before running the current batch.
+        batch_queue_cls: the class to use for the underlying batch queue.
```

Comment:
Is `batch_queue_cls` still necessary? Is there a simple unit test you could write with it?

Reply:
yes, Aviary need it for a priority-based `_BatchQueue`

Reply:
Sounds good.